projects/storefrontapp-e2e-cypress/cypress/support/select-user-menu-option.commands.ts

```diff
@@ -1,6 +1,9 @@
-declare namespace Cypress {
-  interface Chainable {
-    /**
+import { isMobile } from '../helpers/viewport-context';
```

Comment:
Please verify it also on your machine, after building from scratch all libs I am getting VSC TypeScript error here saying `Module "../helpers/viewport-context" has no exported member 'isMobile'`